src/components/Header/UrlInputContainer.jsx

```diff
@@ -31,55 +29,19 @@ function UrlInputContainer({
     const numOfURL = inputValues.length;
 
     if (numOfURL > 30) {
-      textarea.current.value = "";
-      textarea.current.style.height = "auto";
+      textareaRef.current.value = "";
 
       alert("URL 입력은 30개 이하로 해주세요");
 
-      return false;
+      return null;
```

Comment:
```suggestion
      return;
```

함수종료를 위한 반환문은 `return;` 으로 작성하시면 될 것 같습니다!